main.go

```diff
@@ -18,6 +18,7 @@ import (
 	"code.gitea.io/gitea/modules/setting"
 
 	// register supported doc types
+	_ "code.gitea.io/gitea/modules/markup/console"
 	_ "code.gitea.io/gitea/modules/markup/csv"
 	_ "code.gitea.io/gitea/modules/markup/markdown"
 	_ "code.gitea.io/gitea/modules/markup/orgmode"
```

Comment:
I don't thin it does matter as they are exclusive

Reply:
Given that you detect here whether your content contains `\x1b` (or something similar), the order certainly does matter.
Files that do not want to be color coded could also use that char (even though they shouldn't).

Reply:
no this detection will only happen if it's not detected by filename or extension already, so if file is detected already as source markdown/csv/source code highlighting/custom renderer this detection will not even happen. Pretty much it will try to detect only if it would be otherwise rendered as plain text

Reply:
also go fmt will order them alphabetically, also code should not depend on import statement order. if that would be needed it should be done code using some kind of priority field or something like that